documentation/API_documentation/call-forwarding-signal-API-Readiness-Checklist.md

```diff
@@ -1,15 +1,15 @@
 
 # API Readiness Checklist
 
-Checklist for API call-forwarding-signal v0.3.0-rc.1 in r2.1
+Checklist for API call-forwarding-signal v0.3.0 in r2.2
 
 
 | Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference information |
 |----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----|
 |  1 | API definition                               |   M   |         M         |    M    |    M   |   Y  | [link](/code/API_definitions/call-forwarding-signal.yaml) |
 |  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |   Y  | r2.2 |
 |  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |   Y  | r2.2 |

```

Comment:
The public releases should be applied (there are no relevant changes between r2.2 and r2.3 for the CallForwarding API):

```suggestion
|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |   Y  | [r2.3](https://github.com/camaraproject/Commonalities/releases/tag/r2.3) |
|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |   Y  | [r2.3](https://github.com/camaraproject/IdentityAndConsentManagement/releases/tag/r2.3) |

```